benchmarks/benchmark.js

```diff
@@ -289,6 +290,15 @@ if (process.env.PORT) {
     })
   }
 
+  experiments['undici - fetch (with diagnostics channel)'] = () => {
+    diagnosticsChannel.subscribe('undici:fetch:asyncEnd', () => {})
```

Comment:
This essentially create a memory leak and it's never removed. You should do a separate benchmark file.

Reply:
@mcollina so I would be able to resolve the memory leak if I do something like:

```
experiments['undici - fetch (with diagnostics channel)'] = () => {
    const onMessage = () => {}
    diagnosticsChannel.subscribe('undici:fetch:asyncEnd', onMessage)
    return makeParallelRequests(resolve => {
      fetch(dest.url).then(res => {
        res.body.pipeTo(new WritableStream({ write () {}, close () { resolve() } }))
      }).catch(console.log)
    }).finally( () => { diagnosticsChannel.unsubscribe('undici:fetch:asyncEnd', onMessage) })
  }
```

Would you still want this in a different file? If so, with same experiments and should we add all channels?